# Message containing an array of possible robot spawn positions and target positions

geometry_msgs/Pose[]        positions               # Array of Pose messages